apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -417,6 +432,17 @@ function recordAcceptedSheet(store: Store, { interpretation }: Context) {
   if (interpretation.type === 'NeedsReviewSheet') {
     store.adjudicateSheet(sheetId);
   }
+  if (
+    isFeatureFlagEnabled(
+      BooleanEnvironmentVariableName.ENABLE_CONTINUOUS_EXPORT
+    )
+  ) {
+    (
+      await exportCastVoteRecordsToUsbDrive(store, usbDrive, [
+        assertDefined(store.getResultSheet(sheetId)),
+      ])
+    ).unsafeUnwrap();
+  }
```

Comment:
In my testing, I've found that exporting CVRs does not block the display of the success check, which is great! The exporting happens in the background after the success check is displayed.

I do intend to think more deeply about and revisit error handling. For now, if the export fails, you'll see the catch-all "something went wrong, please restart" error screen.

Issue to track: https://github.com/votingworks/vxsuite/issues/3839